exercises/concept/meltdown-mitigation/.docs/introduction.md

```diff
@@ -0,0 +1,82 @@
+# Conditionals
+
+The [conditionals][control flow tools] [`if`][if statement], `elif` (_a contraction of 'else and if'_) and `else` are used in Python to control the flow of execution and make decisions in a program.
+Unlike many other programming langauges, Python versions 3.9 and below do not offer a formal case-switch statement, using multiple `elif` statements to serve a similar purpose.
+Python 3.10 introduces a variant case-switch statement called `pattern matching`, which will be covered in another exercise.
+Conditional statements pair with expressions and objects that must resolve to `True` or `False` -- either by returning a `bool` directly, or by evaluating ["truthy" or "falsy"][truth value testing].
```

Comment:
This paragraph seems very familiar, I think I reviewed it with suggestions in another PR

Reply:
You did.  And **_thank you_** for noting it.  You did have changes there, and I will make them here as well.  😄